server/graphql/schema/ingestion/resolvers/mutations/ingestData.ts

```diff
@@ -752,19 +825,24 @@ class IngestDataWorker extends ResolverBase {
         if (!modelDB)
             modelDB = JCOutput.modelConstellation.Model;
         else {
+            // if a model exists in the database, we need to use its name and title
+            // so that the JCOutput doesn't overwrite it with the asset name
+            const modelDBName = modelDB.Name;
+            const modelDBTitle = modelDB.Title;
             modelDB.cloneData(JCOutput.modelConstellation.Model);
+            modelDB.Name = modelDBName;
+            modelDB.Title = modelDBTitle;
             cloned = true;
         }
 
-        modelDB.Name = itemDB ? NameHelpers.modelDisplayName(model.subtitle, itemDB, subjectsDB) : model.subtitle;
-        modelDB.Title = model.subtitle;
+        if (!updateMode) modelDB.Name = itemDB ? NameHelpers.modelDisplayName(model.subtitle, itemDB, subjectsDB) : model.subtitle;
+        if (!updateMode) modelDB.Title = model.subtitle.length > 1 ? model.subtitle : modelDB.Title;
```

Comment:
If I understand this code correctly, I believe you want this to be `model.subtitle.length > 0`, not `1`

Reply:
I've ran into cases where ':' is sent.
Should I just set it to 0?

Reply:
It seems to me that a subtitle of ':' is probably a bug.  Can you track that down, and fix that, too?  Separately, this code should not be "fixing" that bug produced elsewhere.  Locate fixes with the actual issues, not in patch up logic found elsewhere (such as here)....